image/tools/entrypoint.sh

```diff
@@ -23,6 +23,10 @@ while getopts "c:b:e:d:" opt; do
     esac
 done
 
+if [[ "$debug" ]]; then
+    set -x
```

Comment:
?

Reply:
This will show every executed command, e.g.:
```
$ ./image/tools/entrypoint.sh -c 3scale-redis -e "" -d "true"
+ [[ -z 3scale-redis ]]
+ source /Users/omatskiv/projects/backup-container-image/image/tools/lib/backend/s3.sh
+ [[ -n '' ]]
+ source /Users/omatskiv/projects/backup-container-image/image/tools/lib/component/3scale-redis.sh
++ date +%Y/%m/%d
+ DATESTAMP=2019/03/05
+ DEST=/tmp/intly
+ ARCHIVES_DEST=/tmp/intly/archives
+ mkdir -p /tmp/intly /tmp/intly/archives
+ export HOME=/tmp/intly
+ HOME=/tmp/intly
+ component_dump_data /tmp/intly
+ dest_file=/tmp/intly/archives/dump.rdb
+ dumb_rdb_path=/var/lib/redis/data/dump.rdb
+ oc projects
```

Reply:
I thought this would help when debugging issues in container or locally. You will be able to see what exactly was executed. It even prints expanded commands.